pug-sass/src/mihaelachetran/blog-blocks/index.pug

```diff
@@ -1,16 +1,24 @@
 doctype html
 html(lang="en")
-  head
-    meta(charset="UTF-8")
-    meta(name="viewport", content="width=device-width, initial-scale=1.0")
-    title I am brave, I am Mihaela 009!
-    link(rel="stylesheet", href="index.css")
-  body
-    .main-box
-      .images-container
-        h1.images-title Effective Forms Advertising Internet Web Site
-        p.images-subtitle.images-subtitle Make Money Online Through Advertising
-        .images-avatar
-        .images-name Lora Steven
-        .images-date Aug 15
-        .images-icon
+head
```

Comment:
head and body have to be children of the **html** tag

Reply:
Thank you! Done!